RecoTracker/MkFit/python/mkFitInputConverter_cfi.py

```diff
@@ -4,7 +4,6 @@
 from RecoLocalTracker.SiStripClusterizer.SiStripClusterChargeCut_cfi import *
 
 mkFitInputConverter = _mkFitInputConverterDefault.clone(
-    minGoodStripCharge = cms.PSet(
-        refToPSet_ = cms.string('SiStripClusterChargeCutLoose')
-    )
+    minGoodStripCharge = dict(
+        refToPSet_ = cms.string('SiStripClusterChargeCutLoose'))
```

Comment:
this can also get dropped

Reply:
The aim was to override the existing
```python
mkFitInputConverter.minGoodStripCharge = cms.PSet(
    value = cms.required.double
)
```
with
```python
mkFitInputConverter.minGoodStripCharge = cms.PSet(
    refToPSet_ = cms.string('SiStripClusterChargeCutLoose')
)
```

With @jeongeun's change the resulting PSet would be
```python
mkFitInputConverter.minGoodStripCharge = cms.PSet(
    value = cms.required.double,
    refToPSet_ = cms.string('SiStripClusterChargeCutLoose')
)
```
which should still work properly because `refToPSet_` overrides whatever else exists in the PSet. But I would find it clearer if `refToPSet_` would be the only member of the PSet. Therefore I'd suggest to use `cms.PSet(refToPSet_ = cms.string('...'))` instead of `dict(...)` everywhere a `refToPSet_` is inserted, because the aim is to override the existing PSet instead of overriding a single PSet member.

Dropping `cms.string()` should not work because the original PSet does not have `refToPSet_` member.